index.bs

```diff
@@ -575,68 +575,182 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		<div id="navigator-idl" dfn-for="Navigator">
 
 		<div class="algorithm" data-algorithm="navigator-clipboard">
-		<h4 id="h-navigator-clipboard"><dfn>clipboard</dfn></h4>
-		The [=clipboard=] attribute must return the {{Navigator}}'s [=clipboard=] object.
+		<h4 id="h-navigator-clipboard">clipboard</h4>
+		It is used to execute read/write operations from/to the [=system clipboard=].
+		The {{Navigator/clipboard}} getter steps are to return [=this=]'s [=clipboard=] object.
 		</div>
 
 		</div><!-- dfn-for Navigator -->
 
-	<h3 id="clipboard-interface">Clipboard Interface</h3>
-
+	<h3 id="clipboard-item-interface">ClipboardItem Interface</h3>
 		<pre class="idl" data-highlight="webidl">
-		typedef sequence&lt;ClipboardItem> ClipboardItems;
+			typedef (DOMString or Blob) ClipboardItemDataType;
+			typedef Promise&lt;ClipboardItemDataType&gt; ClipboardItemData;
 
-		[SecureContext, Exposed=Window] interface Clipboard : EventTarget {
-		  Promise&lt;ClipboardItems> read();
-		  Promise&lt;DOMString> readText();
-		  Promise&lt;undefined> write(ClipboardItems data);
-		  Promise&lt;undefined> writeText(DOMString data);
-		};
+			[SecureContext, Exposed=Window]
+			interface ClipboardItem {
+				constructor(record&lt;DOMString, ClipboardItemData> items,
+				            optional ClipboardItemOptions options = {});
 
-		typedef (DOMString or Blob) ClipboardItemDataType;
-		typedef Promise&lt;ClipboardItemDataType> ClipboardItemData;
+				readonly attribute PresentationStyle presentationStyle;
+				readonly attribute FrozenArray&lt;DOMString> types;
 
-		callback ClipboardItemDelayedCallback = ClipboardItemData ();
+				Promise&lt;Blob&gt; getType(DOMString type);
+			};
 
-		[Exposed=Window] interface ClipboardItem {
-		  constructor(record&lt;DOMString, ClipboardItemData> items,
-		    optional ClipboardItemOptions options = {});
-		  static ClipboardItem createDelayed(
-		      record&lt;DOMString, ClipboardItemDelayedCallback> items,
-		      optional ClipboardItemOptions options = {});
+			enum PresentationStyle { "unspecified", "inline", "attachment" };
 
-		  readonly attribute PresentationStyle presentationStyle;
-		  readonly attribute long long lastModified;
-		  readonly attribute boolean delayed;
+			dictionary ClipboardItemOptions {
+				PresentationStyle presentationStyle = "unspecified";
+			};
+		</pre>
 
-		  readonly attribute FrozenArray&lt;DOMString> types;
+		<p><dfn for="ClipboardItem">Clipboard Item</dfn></p>
 
-		  Promise&lt;Blob> getType(DOMString type);
-		};
+		A [=clipboard item=] is conceptually data that the user has expressed a desire to make shareable by invoking a "cut" or "copy" command.
+		For example, if a user copies a range of cells from a spreadsheet of a native application, it will result in one [=clipboard item=]. If a user copies a set of files from their desktop, that list of files will be represented by multiple [=clipboard item=]s.
```

Comment:
So, here I'm describing the conceptual `clipboard item` object's model. This is not relevant to web developers so not sure if this belongs to an Example section.

Reply:
> So, here I'm describing the conceptual `clipboard item` object's model. This is not relevant to web developers so not sure if this belongs to an Example section.

For the general understanding of web developers about the async clipboard API, it seems relevant, see also https://github.com/w3c/clipboard-apis/pull/158#discussion_r747810984.

I suggest moving most of the prose of this section, including "A clipboard item is conceptually data that the user has expressed a desire to make shareable by invoking a "cut" or "copy" command. For example, if a user copies a range of cells from a spreadsheet of a native application, it will result in one clipboard item. If a user copies a set of files from their desktop, that list of files will be represented by multiple clipboard items. Some platforms may support having more than one clipboard item at a time on the Clipboard, while other platforms replace the previous clipboard item with the new one." to a note (https://respec.org/docs/#note) of the `Clipboard` (not `ClipboardItem`) interface.

"A clipboard item has a list of representations, each representation with an associated mime type and data. In the example where the user copies a range of cells from a spreadsheet, it may be represented as an image (image/png), an HTML table (text/html), or plain text (text/plain). Each of these mime types describe a different representation of the same clipboard item at different levels of fidelity and make the clipboard item more consumable by target applications during paste. Making the range of cells available as an image will allow the user to paste the cells into a photo editing app, while the text/plain format can be used by text editor apps.

A clipboard item can also optionally have a presentation style that helps distinguish whether apps "pasting" a clipboard item should insert the contents of an appropriate representation inline at the point of paste or if it should be treated as an attachment." would be better placed as a note of the `ClipboardItem`interface.

This section should then only contain the definition of the conceptual `clipboard item` and the definition of the `constructor`.

Reply:
> would be better placed as a note of the ClipboardIteminterface.

I disagree? That text is defining the structure of the conceptual `clipboard item`, which seems pretty important. It shouldn't be a note (since notes are non-normative), and it's  only indirectly related to the `ClipboardItem` interface (since a `ClipboardItem` contains a conceptual `clipboard item`).

Similarly the first paragraph you mention,

> to a note (https://respec.org/docs/#note) of the Clipboard (not ClipboardItem) interface.

should not be a note. I don't know why it would be more related to `Clipboard` than `ClipboardItem`, either...

Reply:
> > would be better placed as a note of the ClipboardIteminterface.
> 
> I disagree? That text is defining the structure of the conceptual `clipboard item`, which seems pretty important. It shouldn't be a note (since notes are non-normative), and it's only indirectly related to the `ClipboardItem` interface (since a `ClipboardItem` contains a conceptual `clipboard item`).
> 

Thanks for the comment. I didn't express myself clearly, sorry. I suggest separating the examples and the definition of the `clipboard item` concept. Wouldn't the examples be better located in a note of the `ClipboardItem` interface, intended for web-developers? 

> Similarly the first paragraph you mention,
> 
> > to a note (https://respec.org/docs/#note) of the Clipboard (not ClipboardItem) interface.
> 
> should not be a note. I don't know why it would be more related to `Clipboard` than `ClipboardItem`, either...

It's about the level of abstraction of the examples: the examples for web-developers about multiple clipboard items should belong to the `Clipboard` interface, because only that exposes multiple items. The examples for one clipboard item should belong to the `ClipboardItem` interface.

WDYT? My understanding of the distinction between the conceptual clipboard item and `ClipboardItem` might still be wrong.